src/App.js

```diff
@@ -21,6 +22,7 @@ function App() {
         <Route path="/wishlist" element={<WishlistPage />} />
         <Route path="/login" element={<UserLogin />} />
         <Route path="/signup" element={<SignUp />} />
+        <Route path="/product/:productId" element={<SingleProduct />} />
```

Comment:
you can omit the `/` in the path of a route. not needed in react router v6

Reply:
Thanks , I was unaware of this .